stream-parent/stream-client/src/test/java/org/interledger/stream/sender/SimpleStreamSenderTests.java

```diff
@@ -65,7 +66,13 @@ public void constructThreeArgWithNullLink() {
   @Test
   public void constructThreeArgWithNullExecutor() {
     expectedException.expect(NullPointerException.class);
-    new SimpleStreamSender(streamEncryptionServiceMock, linkMock, null);
+    new SimpleStreamSender(streamEncryptionServiceMock, linkMock, (ExecutorService) null);
+  }
+
+  @Test
+  public void constructThreeArgWithNullSleep() {
```

Comment:
Not sure if it's worth it, but should we make a unit test with a simulated Clock to test the loop?

Reply:
Check out the latest commit to `SimpleStreamSenderIT`.  I send the same payment through a `SimpleStreamSender` with sleep time set to 100ms and a `SimpleStreamSender` with sleep time set to 5ms.  Total time to send the payments (in this case 7 packets) was much much lower with a smaller sleep time

Reply:
Per the ILP community call, consensus is that this is probably not worth the effort.